cmd/restic/cmd_cat.go

```diff
@@ -42,10 +42,13 @@ func runCat(gopts GlobalOptions, args []string) error {
 		return err
 	}
 
-	lock, err := lockRepo(gopts.ctx, repo)
-	defer unlockRepo(lock)
-	if err != nil {
-		return err
+	if !gopts.NoLock {
+		lock, err := lockRepo(gopts.ctx, repo)
+		// Make the linter happy
+		defer func() { _ = unlockRepo(lock) }()
+		if err != nil {
```

Comment:
The error check should happen before the `defer unlock()`, apart from that it's fine.

Reply:
Done.